server/test/categoryRouter.test.js

```diff
@@ -0,0 +1,74 @@
+const Category = require('../db/models/category');
+const server = require('../index');
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const { getMongoDB } = require('../db');
+const { MongoMemoryServer } = require('mongodb-memory-server');
+const mongoose = require('mongoose');
+const User = require('../db/models/user');
+
+let exampleCategory;
+let defaultUser;
+let mongoServer;
+const expect = chai.expect;
```

Comment:
Curious, why expect over assert?

Reply:
No particular reason! It was probably because all the tutorials I saw were using expect. We can change it if you like